recipes/sonarqube/index.js

```diff
@@ -1,6 +1,6 @@
 module.exports = Ferdium =>
   class SonarQube extends Ferdium {
     buildUrl(url) {
-      return `${url}/`;
+      return `${url}`;
```

Comment:
What's the point of defining this function? Please remove.

Reply:
Fixed on other PR as well, excuse me for the delay :)